client/state/sites/plans/actions.js

```diff
@@ -127,7 +127,7 @@ export function fetchSitePlansCompleted( siteId, data ) {
 	let plansData = omit( data, '_headers' );
 	const { product_id } = personalPlan;
 
-	if ( ! has( plansData, product_id ) ) {
+	if ( ! isEmpty( plansData ) && ! plansData.hasOwnProperty( product_id ) ) {
```

Comment:
it's cool to go back to this and all, but did you intend on it? looks like this could have come in through a merge conflict